src/components/SoraCard/steps/RoadMap.vue

```diff
@@ -48,53 +48,86 @@
       @click="handleConfirm"
     >
-      <span class="text">{{ btnText() }}</span>
-      <s-icon name="arrows-arrow-top-right-24" size="18" class="" />
+      <span class="text">{{ btnText }}</span>
+      <s-icon v-if="showArrow" name="arrows-arrow-top-right-24" size="18" />
     </s-button>
+    <notification-enabling-page v-if="permissionDialogVisibility">
+      {{ t('code.allowanceRequest') }}
+    </notification-enabling-page>
   </div>
 </template>
 
 <script lang="ts">
 import { loadScript, unloadScript } from 'vue-plugin-load-script';
+import { action, getter, state } from '@/store/decorators';
 import TranslationMixin from '@/components/mixins/TranslationMixin';
-import { Component, Mixins } from 'vue-property-decorator';
+import { Component, Mixins, Prop } from 'vue-property-decorator';
 import EmailIcon from '@/assets/img/sora-card/email.svg?inline';
 import CardIcon from '@/assets/img/sora-card/card.svg?inline';
 import UserIcon from '@/assets/img/sora-card/user.svg?inline';
 import { delay } from '@/utils';
-import { mixins } from '@soramitsu/soraneo-wallet-web';
+import { clearTokensFromSessionStorage } from '@/utils/card';
+import { mixins, components } from '@soramitsu/soraneo-wallet-web';
+import { CardIssueStatus } from '@/types/card';
+import { XOR } from '@sora-substrate/util/build/assets/consts';
 
 @Component({
   components: {
     EmailIcon,
     CardIcon,
     UserIcon,
+    NotificationEnablingPage: components.NotificationEnablingPage,
   },
 })
-export default class RoadMap extends Mixins(TranslationMixin, mixins.LoadingMixin) {
+export default class RoadMap extends Mixins(
+  TranslationMixin,
+  mixins.LoadingMixin,
+  mixins.CameraPermissionMixin,
+  mixins.TranslationMixin
+) {
+  @state.soraCard.userKycStatus private userKycStatus!: CardIssueStatus;
+  @getter.soraCard.isEuroBalanceEnough isEuroBalanceEnough!: boolean;
+  @action.soraCard.getUserKycStatus private getUserKycStatus!: AsyncFnWithoutArgs;
+
+  @Prop({ default: false, type: Boolean }) readonly userApplied!: boolean;
+
   firstPointChecked = false;
   firstPointCurrent = true;
   secondPointChecked = false;
   secondPointCurrent = false;
   thirdPointChecked = false;
   thirdPointCurrent = false;
+  showArrow = true;
 
+  btnText = 'LET’S START';
+  btnDisabled = false;
   btnLoading = false;
 
-  btnText(): string {
-    if (sessionStorage.getItem('access-token')) {
-      return 'FINISH THE KYC';
-    }
-    return 'LET’S START';
-  }
+  permissionDialogVisibility = false;
 
   async handleConfirm(): Promise<void> {
+    try {
+      const mediaDevicesAllowance = await this.checkMediaDevicesAllowance('SoraCard');
+
+      if (!mediaDevicesAllowance) return;
+    } catch (error) {
+      console.error('[KYC Sora Card]: Camera error.', error);
+    }
+
     if (sessionStorage.getItem('access-token')) {
-      this.$emit('confirm-start');
+      await this.getUserKycStatus();
+
+      if (this.userKycStatus === CardIssueStatus.Success) {
```

Comment:
`this.$emit('confirm-start-kyc', this.userKycStatus !== CardIssueStatus.Success);` instead of if-else